test/src/test/java/hudson/model/ParametersTest.java

```diff
@@ -73,7 +73,7 @@ public void parameterTypes() throws Exception {
         System.out.println(o);
         HtmlCheckBoxInput booleanParameterInput = (HtmlCheckBoxInput) o;
         assertEquals(true, booleanParameterInput.isChecked());
-        assertEquals("boolean", ((HtmlElement) DomNodeUtil.selectSingleNode(element, "td[@class='setting-name']")).getTextContent());
+        assertEquals("boolean", ((HtmlElement) DomNodeUtil.selectSingleNode(element, "td[@class='setting-main']")).getTextContent());
```

Comment:
Why does it happen?

Reply:
In the old style, you have something roughly like:

setting-name | setting-main
------------ | -------------
Label | [x]

In the new, it's closer to:

setting-name | setting-main
------------ | -------------
~ | [x] Label